store.py

```diff
@@ -216,46 +409,200 @@ def accept(self, customer: Customer) -> bool:
         >>> line.queue == [c1]
         True
         """
+        if self.can_accept(customer):
+            self.queue.append(customer)
+            return True
+        return False
 
     def start_checkout(self) -> int:
         """Checkout the next customer in this CheckoutLine.
-
         Return the time it will take to checkout the next customer.
+        Assume that there is a customer in line when this is called.
         """
+        raise NotImplementedError
 
     def complete_checkout(self) -> bool:
         """Finish the checkout for this CheckoutLine.
 
         Return whether there are any remaining customers in the line.
+        Customer is in the queue until they are done checking out.
+        >>>
         """
+        self.queue.pop(0)
+        return len(self.queue) >= 1
 
     def close(self) -> List[Customer]:
         """Close this line.
 
         Return a list of all customers that need to be moved to another line.
+        The last person in line will join another queue (first) as the line
+        closes, and the rest of the people will join each second after.
         """
+        i = 0
+        result = []
+        reverse_queue = self.queue[::-1]
+        while i != (len(self.queue) - 1):
+            to_move = reverse_queue.pop(i)
+            result.append(to_move)
+            i += 1
+        return result
```

Comment:
why complexity  ? I think you are reversing the queue twice here. Also while is not needed. your queue does not change. Use determenistic "for"